lib/filesystem.nix

```
@@ -42,4 +42,16 @@
               type = (builtins.readDir parent).${base} or null;
           in file == /. || type == "directory";
     in go (if isDir then file else parent);
+
+
+  # readTree: Path -> [ Path ]
+  #
+  # Given a directory, return a flattened list of all files within it.
+  readTree = dir: lib.flatten(lib.mapAttrsToList(name: type:
+    if (type == "directory") then
+      lib.filesystem.readTree(dir + "/${name}")
```

```suggestion
  readTree = dir: lib.flatten (lib.mapAttrsToList (name: type:
    if type == "directory" then
      lib.filesystem.readTree (dir + "/${name}")
```

To make the style consistent